server/controllers/reportControllers.js

```diff
@@ -1,28 +1,47 @@
-/* eslint-disable camelcase */
-/* eslint-disable no-undef */
-/* eslint-disable sort-keys */
 import {
     CREATED, getStatusText, INTERNAL_SERVER_ERROR, NOT_FOUND, OK
 } from 'http-status-codes';
+import dotenv from 'dotenv';
+import sendMail from '../middlewares/nodemailer';
+
+dotenv.config();
+
+const {
+    MAP_API_KEY,
+} = process.env;
 
 const db = require('./promise').ReportDb;
 
+const messageBody = (lat, lng, link) => `<b>Emergency </b> <br></br><br></br>
+    <img src="https://maps.googleapis.com/maps/api/staticmap?
+    center=${lat},${lng}
+    &zoom=20&size=500x500
+    &markers=color:red%7Clabel:S%7C${lat},${lng}
+    &key=${MAP_API_KEY}"> <br></br><br></br> 
+    ${link}`;
+
 const Reports = {
     async create(req, res) {
         const { public_id: audioId, url: audioUrl } = await req.file;
         const queryText = { ...req.body, audioId, audioUrl };
+        const { coords } = queryText;
 
         try {
             const report = await db.create(queryText);
+            sendMail('lifeaidd@gmail.com',
+                'Emergency report',
+                'faithomojola@yahoo.com',
+                messageBody(coords[0], coords[1], audioUrl));
```

Comment:
this should be `messageBody(coords, audioUrl)`. Let the function determine how to use the coords that you pass it

Reply:
ok